api/DEPLOYMENT.md

```diff
@@ -79,3 +79,27 @@ e.g. `../.local-storage/network/mainnet.json`
 ```
 
 An optional `permaNodeEndpoint` can be added if you have access to a chronicle node.
+
+## Running with Docker
+
+A Dockerfile is also provided so that you can run the API as a Docker container. After configuring the API as explained in the [Configuration](#Configuration) section:
```

Comment:
Maybe I didn't explain myself, it would be good to show an example of where you locate the .json files for configuration - I don't think the default .local-storage would work in this case

Reply:
yes, very true it was incomplete. I hope now is complete and understandable.